documentation/applying_manifest_blocks.md

```diff
@@ -403,97 +403,93 @@ Create a manifest that sets up a web server with nginx, and run it as a plan.
+1. Add the `puppetlabs-iis` module to your `bolt-project.yaml` under the `modules` key:
+   ```
+   modules:
+     - name: puppetlabs-iis
+       version_requirement: 4.3.2
+   ```
+1. Run `bolt module install` or `Install-BoltModule` if you're using PowerShell.
+1. Create a new Puppet language Bolt plan inside your project by running
+   `bolt plan new profiles` or `New-BoltPlan profiles --pp` on PowerShell . This creates a `plans/`
+   directory in your Bolt project, and a file named `init.pp` inside that directory with some basic
+   commands.
+1. Replace the contents of `<PROJECT DIRECTORY>/plans/init.pp` with this Bolt plan: 
+   ```puppet
+   plan profiles(
+        TargetSpec $targets,
+        String $site_content = 'hello!',
+   ) {
+
+     # Install the puppet-agent package if Puppet is not detected. 
+     # Copy over custom facts from the Bolt modulepath.
+     # Run the `facter` command line tool to gather target information.
+     $targets.apply_prep
+
+     # Compile the manifest block into a catalog
+     return apply($targets, '_catch_errors' => true) {
+       $iis_features = ['Web-WebServer','Web-Scripting-Tools']
+
+       iis_feature { $iis_features:
+         ensure => 'present',
+       }
 
-           file { 'minimal':
-             ensure => 'directory',
-             path   => 'c:\\inetpub\\minimal',
-           }
+       # Delete the default website to prevent a port binding conflict.
+       iis_site {'Default Web Site':
+         ensure  => absent,
+         require => Iis_feature['Web-WebServer'],
+       }
 
-           file { 'content':
-             ensure  => 'file',
-             path    => 'c:\\inetpub\\minimal\\index.html',
-             content => $site_content,
-           }
-         }
+       iis_site { 'minimal':
+         ensure          => 'started',
+         physicalpath    => 'c:\\inetpub\\minimal',
+         applicationpool => 'DefaultAppPool',
+         require         => [
+           File['minimal'],
+           Iis_site['Default Web Site']
+         ],
        }
-    ```
-1.  Run the plan on a target:
 
-    ```shell
-    bolt plan run profiles::iis_install --targets mytarget.mydomain --transport winrm
-    ```
+       file { 'minimal':
+         ensure => 'directory',
+         path   => 'c:\\inetpub\\minimal',
+       }
 
+       file { 'content':
+         ensure  => 'file',
+         path    => 'c:\\inetpub\\minimal\\index.html',
+         content => $site_content,
+       }
+     }
+   }
+   ```
+   This plan uses the `apply` function to apply a manifest to targets. The manifest uses the 
+   [puppetlabs-iis module](https://forge.puppet.com/modules/puppetlabs/iis) to set up an IIS 
+   server on the targets.
+1. Run the plan on a target:
+   _PowerShell cmdlet_
+   ```shell
+   Invoke-BoltPlan profiles --targets winrm://mytarget.mydomain
+   ```
+
+   _\*nix shell command_
+   ```shell
+   bolt plan run profiles --targets winrm://mytarget.mydomain
+   ```
```

Comment:
Small nitpick, but we should be consistent with the order of the shell command & PS cmdlet. Here and in the first bullet point above, the PS cmdlet comes first, whereas the second and third bullet points have the shell command first.